### ISSUE TITLE
[Suggestion] AI Modes

### ISSUE THREAD

**Issue by Sofa499** [tsunamayo/Starship-EVO#2973]
At present, it can be a bit difficult to stage fights with enemy AI. They tend to jump away and attack stations and NPC ships, and if I want to test a new ship's firepower or just do some PVE with friends, I sometimes have to go on a wild goose-chase to find the enemy ship I spawned.

It may not be worth Tsuna's time in this placeholder period of development, but it could be useful to divide spawnable AI into a few different modes:

Enemy
 **- Raider Enemy:** Will aggressively hunt down NPC ships and stations. Similar to current enemy AI.
 **- Static Enemy:** Will prefer to hold its position not attempt to lightcruise anywhere. For if you want enemies guarding something.
 **- Hunter Enemy:** Will aggressively hunt down the player's ship. Also similar to current enemy AI, but disregards NPCs in favor of players.

Friendly (if / when possible)
 **- Escort Ally:** Will attempt to follow the player and aggressively engage hostile ships.
 **- Wingman Ally:** Will attempt to follow the player and engage enemies, but will not break formation with the player's ship.
 **- Static Ally:** Will prefer to hold its position and not attempt to lightcruise anywhere.

The above could make for a number of entertaining possibilities, such as staging fleet battles, or allowing a player to act as a "Game Master" by spawning different enemy NPCs in multiplayer for their friends to fight. In addition to building, I think it could effectively tide people over while the real game takes shape.

**Comment by matejkar** [comment]
10000000% agree

**Comment by StarrKiss** [comment]
Sounds cool I like it. @tsunamayo  plz

**Comment by tsunamayo** [comment]
Yes it is lame atm... Will get back on this as soon as finish the holomap / galaxy (it is taking forever...)

**Comment by matejkar** [comment]
@tsunamayo does holomap mean we will see actual planets with actual landscapes in the game which we can visit?

**Comment by StarrKiss** [comment]
@matejkar  planets are a long way off- holomap means just galaxies/maps. Not that there's anything actually there- as far as I know

**Comment by tsunamayo** [comment]
yeah no sorry planets are a long way off...
